### PR TITLE
Add more to readme

### PR DESCRIPTION
**Pull request by edulop91**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,27 +23,32 @@ Binaries are available on the [releases](https://github.com/chanzuckerberg/bless
...
+
 #### Init
 
 `blessclient init` will ask you some questions in order to assemble some basic configuration.
 
 #### Import-config
 Alternatively, you can also use pre-generated config files.
 
-`blessclient import-config http://github.com/..../teamA_blessclient.yml`
+`blessclient import-config git@github.com:/..../teamA/blessconfig.yml`
```
Comment (ryanking) [inline README.md]: can we maybe give a few examples here, so that it is clear that it is not just git+ssh?

File: README.md [hunk 2]
```diff
@@ -69,3 +74,8 @@ This ssh config does a couple of interesting things -
 
 ## Telemetry
 There currently is some basic trace instrumentation using [honeycomb](https://www.honeycomb.io/). We use this internally to track usage, gather performance statistics, and error reporting. Telemetry is disabled without a honeycomb write key - which you must provide through the [config](pkg/config/config.go).
+
+## Common Errors
+
+### Unsafe RSA public key
```
Comment (ryanking) [inline README.md]: can the error message link to this?

Comment (edulop91) [reply README.md]: I kind of want to write better error handling logic/messaging and not just this readme

Comment (edulop91) [reply README.md]: probably in a separate pr

**Pull request merged**